### DISCUSSION TITLE
Transitor based LLC

### DISCUSSION THREAD

**Discussion by Richard7619** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#544] [Q&A] [answered]
Hi,

In the BOM it includes a Transitor based LLC (the red one). Can anyone share a link for where to buy this? Or what to google to find. 
I know it says may not be needed but if it removes a potential issue I may get anyway. 
Thanks,

**Comment by visualapproach** [comment, marked as answer]
Search on eBay or google for

8-channel bidirectional logic level shifter module

**Reply by Richard7619** [reply]
thanks- I have the S100102 which is a 2021+ model. 
It says on some 2021+ models you will need the red LLC (or no LLC) and PCB_V2B.
Do I still need to order the TXS0108E (the blue one) or is that just an alternative for older models? 
Looking at the finished PCB there seems only to be room for either the blue or red and you wouldn't need to order both. So I think I've answered my own question and I'll just order the red one. 👍 
Thanks,

**Reply by Richard7619** [reply]
Actually I've just looked at the schematic and is the Level-shifter-8ch piggy backed on top of the TXS0108E if it's used. So the TXS is always used?

**Reply by visualapproach** [reply]
No either blue OR red. Stick to the red.

**Reply by Richard7619** [reply]
great, thanks for confirming.

**Reply by Alexd911** [reply]
Hey, 

Could you share with me how you wired the LLC to the Wemos D1 mini? Im totallly strugling here as I cant find any pics of it.

Also what model/part number I need for the LLC.

Cheers
Alex

**Reply by visualapproach** [reply]
> Could you share with me how you wired the LLC to the Wemos D1 mini? Im totallly strugling here as I cant find any pics of it.

No because there is no pictures of that. Either follow the manual or experiment with the D1 mini, but then you'd have to look at the schematics, understand it and translate to what you are doing.

> Also what model/part number I need for the LLC.

Richard7619 voted my first post above as "answered" . I think you can too.

**Reply by Richard7619** [reply]
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/135166611/6361ad3b-d925-49eb-ad8f-3023767b15ee)![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/135166611/ae1cc313-b7d7-40c7-8c59-07ad1ba6b589)

This is the photo of mine. Note I wired the coloured wires per the extra information in this project. I confess I then went on holiday so I haven't actually finished it yet.

**Reply by Alexd911** [reply]
Hey all,

Thanks Richard for the Pics. I actaully don't have any of those boards, just the D1 Mini V4 which apparantly will not work without the LLC.
Looks ike that would be a nightmare to wire that to the pins on the board I have, so I presume I would have to buy that whole kit which I think you can buy from one gentlaman here.

So probably have to shelf the project or then buy the kit.

Thanks to everyone for helping.

Alex
